akka-stream/src/main/scala/akka/stream/impl/fusing/IteratorInterpreter.scala

```diff
@@ -145,7 +145,8 @@ private[akka] class IteratorInterpreter[I, O](val input: Iterator[I], val ops: S
       outHandlers,
       logics,
       (_, _, _) ⇒ throw new UnsupportedOperationException("IteratorInterpreter does not support asynchronous events."),
-      fuzzingMode = false)
+      fuzzingMode = false,
+      null)
```

Comment:
I have a bench, that is what I used back then to measure this https://github.com/akka/akka/blob/master/akka-bench-jmh/src/main/scala/akka/stream/InterpreterBenchmark.scala (it was slow after I fixed some bugs, and I did all the tricks here to make it fast again. That benchmark hosted the old interpreter as well for comparison). I did a lot of docs (well, could be more, but compared to other internal code, I think it is quite good ;) ): 

https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala#L271
https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala#L68

And there are many more comments in the code itself. I did not explicitly document the use of nulls, but the effort for documentation was there, especially because you asked for more docs before so I delivered :)